geode-core/src/test/java/org/apache/geode/distributed/internal/DistributionConfigJUnitTest.java

```diff
@@ -138,19 +135,19 @@ public void testGetAttributeNames() {
 
     // TODO - This makes no sense. One has no idea what the correct expected number of attributes
     // are.
-    assertEquals(36, boolList.size());
-    assertEquals(35, intList.size());
-    assertEquals(88, stringList.size());
-    assertEquals(5, fileList.size());
-    assertEquals(5, otherList.size());
+    assertThat(boolList).hasSize(36);
+    assertThat(intList).hasSize(35);
+    assertThat(stringList).hasSize(88);
+    assertThat(fileList).hasSize(5);
+    assertThat(otherList).hasSize(5);
   }
 
   @Test
   public void testAttributeDesc() {
     String[] attNames = AbstractDistributionConfig._getAttNames();
     for (String attName : attNames) {
-      assertTrue("Does not contain description for attribute " + attName,
-          AbstractDistributionConfig.dcAttDescriptions.containsKey(attName));
+      assertThat(AbstractDistributionConfig.dcAttDescriptions.containsKey(attName))
+          .as("Does not contain description for attribute " + attName).isTrue();
```

Comment:
Many assertions still need further change after using the "Convert Assertions to AssertJ" IntelliJ plugin. Assertions on collections usually require some further help.

Let's change this assertion to:
```
assertThat(AbstractDistributionConfig.dcAttDescriptions).containsKey(attName);
```